ci/job.yml

```diff
@@ -63,14 +62,13 @@
         https://github.com/centos/container-pipeline-service.
```

Comment:
Is the trailing `.` by mistake or on purpose? I understand it's not a part of PR but asking just to be sure.

Reply:
this is part of the job documentation, so `.` is on purpose